### PR TITLE
Update utility.py

### PR DESCRIPTION
**Pull request by kcwebby**
Proposed edits (untested) to issue #50 
Configuration update to read redis server information from configuration file.

### CODE REVIEW

**Review by dreinhold — COMMENTED** [review]
Should work with a few updates.

File: radio/utility.py [hunk 1]
```diff
@@ -3,8 +3,9 @@
 class RedisQueue(object):
     """Simple Queue with Redis Backend"""
     def __init__(self, name, namespace='tp', **redis_kwargs):
-       """The default connection parameters are: host='localhost', port=6379, db=0"""
-       self.__db= redis.Redis(**redis_kwargs)
+       """Defaults to reading REDIS_URL from environ, or channel REDIS URL from settings_local.py"""
+       channel_redis_url = settings.CHANNEL_LAYERS['default']['CONFIG']['hosts'][0]
```
[the remark this answers is not in the document]
Comment (kcwebby) [reply radio/utility.py:8]: Take a look at that change, is that how you would do it?  I can spin up another machine to test this.

**Comment by dreinhold** [comment]
Just fix your settings import to
`from django.conf import settings`

And that should work.

File: radio/utility.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (kcwebby) [reply radio/utility.py:9]: Like that?  that is new to me.   I should have an environment to test in shortly.

**Comment by kcwebby** [comment]
Random side comment, this doesn't cover nearly as much as I thought.
if redis is not installed and running on localhost, you cannot ./manage.py runserver, as its looks exclusively for localhost, ignoring both settings.py, and settings_local.py.   the exception that is generated, doesn't reference any trunk-player code at all, so is this in django itself?

````
(ASTP)ubuntu@XXXXX:~/trunk-player$ REDIS_URL=redis://trunkXXXXXX.amazonaws.com:6379
(ASTP)ubuntu@XXXXX:~/trunk-player$ ./manage.py runserver 0.0.0.0:80
Performing system checks...

System check identified no issues (0 silenced).
October 06, 2020 - 14:39:25
Django version 1.11.17, using settings 'trunk_player.settings'
Starting Channels development server at http://0.0.0.0:80/
Channel layer default (asgi_redis.core.RedisChannelLayer)
Quit the server with CONTROL-C.
2020-10-06 14:39:25,374 - INFO - worker - Listening on channels http.request, websocket.connect, websocket.disconnect, websocket.receive
Exception in thread Thread-1:
Traceback (most recent call last):
  File "/home/ubuntu/trunk-player/env/lib/python3.8/site-packages/redis/connection.py", line 484, in connect
    sock = self._connect()
  File "/home/ubuntu/trunk-player/env/lib/python3.8/site-packages/redis/connection.py", line 541, in _connect
    raise err
  File "/home/ubuntu/trunk-player/env/lib/python3.8/site-packages/redis/connection.py", line 529, in _connect
    sock.connect(socket_address)
ConnectionRefusedError: [Errno 111] Connection refused

During handling of the above exception, another exception occurred:


```

**Comment by kcwebby** [comment]
So unsure if this PR will work, but did notice that if a local box does NOT have a localhost version of REDIS running, then trunk-player will not answer requests and results in the most vague non-error logging HTTP 400 error ever.

I have a configuration box setup, configured with the above pull request incorporated, and with local redis-server running it works fine (although, because its not pointed at the same redis server, the queue reloaded on the websocket is affected).

If you stop the redis-server on the webserver, and instead rely on the remote redis server, you get the 400 error with this exception.   I will open this back up as an issue.

```
During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/ubuntu/trunk-player/env/lib/python3.8/site-packages/redis/client.py", line 666, in execute_command
    connection.send_command(*args)
  File "/home/ubuntu/trunk-player/env/lib/python3.8/site-packages/redis/connection.py", line 609, in send_command
    self.send_packed_command(self.pack_command(*args))
  File "/home/ubuntu/trunk-player/env/lib/python3.8/site-packages/redis/connection.py", line 584, in send_packed_command
    self.connect()
  File "/home/ubuntu/trunk-player/env/lib/python3.8/site-packages/redis/connection.py", line 488, in connect
    raise ConnectionError(self._error_message(e))
redis.exceptions.ConnectionError: Error 111 connecting to localhost:6379. Connection refused.

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/ubuntu/trunk-player/env/lib/python3.8/site-packages/redis/connection.py", line 483, in connect
    sock = self._connect()
  File "/home/ubuntu/trunk-player/env/lib/python3.8/site-packages/redis/connection.py", line 540, in _connect
    raise err
  File "/home/ubuntu/trunk-player/env/lib/python3.8/site-packages/redis/connection.py", line 528, in _connect
    sock.connect(socket_address)
ConnectionRefusedError: [Errno 111] Connection refused

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/ubuntu/trunk-player/env/lib/python3.8/site-packages/daphne/http_protocol.py", line 162, in process
    self.factory.channel_layer.send("http.request", {
  File "/home/ubuntu/trunk-player/env/lib/python3.8/site-packages/asgi_redis/core.py", line 124, in send
    self.chansend(
  File "/home/ubuntu/trunk-player/env/lib/python3.8/site-packages/redis/client.py", line 2943, in __call__
    return client.evalsha(self.sha, len(keys), *args)
  File "/home/ubuntu/trunk-player/env/lib/python3.8/site-packages/redis/client.py", line 2078, in evalsha
    return self.execute_command('EVALSHA', sha, numkeys, *keys_and_args)
  File "/home/ubuntu/trunk-player/env/lib/python3.8/site-packages/redis/client.py", line 672, in execute_command
    connection.send_command(*args)
  File "/home/ubuntu/trunk-player/env/lib/python3.8/site-packages/redis/connection.py", line 609, in send_command
    self.send_packed_command(self.pack_command(*args))
  File "/home/ubuntu/trunk-player/env/lib/python3.8/site-packages/redis/connection.py", line 584, in send_packed_command
    self.connect()
  File "/home/ubuntu/trunk-player/env/lib/python3.8/site-packages/redis/connection.py", line 488, in connect
    raise ConnectionError(self._error_message(e))
redis.exceptions.ConnectionError: Error 111 connecting to localhost:6379. Connection refused.

2020-10-06 20:12:50,014 Traceback (most recent call last):
  File "/home/ubuntu/trunk-player/env/lib/python3.8/site-packages/redis/connection.py", line 483, in connect
    sock = self._connect()
  File "/home/ubuntu/trunk-player/env/lib/python3.8/site-packages/redis/connection.py", line 540, in _connect
    raise err
  File "/home/ubuntu/trunk-player/env/lib/python3.8/site-packages/redis/connection.py", line 528, in _connect
    sock.connect(socket_address)
ConnectionRefusedError: [Errno 111] Connection refused
```

**Pull request closed without merging**